sensing/radar_scan_to_pointcloud2/schema/radar_scan_to_pointcloud2.schema.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Radar Scan to PointCloud2 Node",
+  "type": "object",
+  "definitions": {
+    "radar_scan_to_pointcloud2": {
+      "type": "object",
+      "properties": {
+        "publish_amplitude_pointcloud": {
+          "type": "boolean",
+          "description": "Whether publish radar pointcloud whose intensity is amplitude.",
+          "default": true
```

Comment:
Please use quotes for all `"default"` values, e.g.: `true` --> `"true"` or else it doesn't auto-populate the parameter file correctly. I missed this in my first review, sorry about that.

This applies to all parameters which have a `"default"` value.

Reply:
Done.